src/components/ApplicantForm/ApplicantForm.tsx

```diff
@@ -105,40 +132,70 @@ const makeTags = (tags:Array<{attribute:string, id:number}>) => {
 							}}
 							/>
 						</label></h6>
-						<input id="username" className="applicant-input" placeholder={username} disabled/>
-						<label htmlFor="username">Refresh to find your perfect anonymized codename</label>
+						<textarea
+							disabled
+							id="username"
+							className="applicant-input"
+							placeholder={username}
+							rows={1}
+						/>
+						<label htmlFor="username" className="privacy-notice">Refresh to find your perfect codename</label>
 						<h6><label htmlFor="bio">Bio</label></h6>
-						<input id="bio" className="applicant-input"
+						<textarea
+							id="bio"
+							className="applicant-input"
 							onChange={handleFormChange}
+							rows={4}
+							maxLength={250}
 						/>
-						<p>The following information will not be shared with employers:</p>
+						<p className="privacy-notice">The following information will not be shared with employers:</p>
 						<h6><label htmlFor="first_name">First Name</label></h6>
-						<input id="first_name" className="applicant-input" 
+						<textarea
+							id="first_name"
+							className="applicant-input" 
 							onChange={handleFormChange}
+							rows={1}
 						/>
 						<h6><label htmlFor="last_name">Last Name</label></h6>
-						<input id="last_name" className="applicant-input" 
+						<textarea
+							id="last_name"
+							className="applicant-input" 
 							onChange={handleFormChange}
+							rows={1}
 						/>
 						<h6><label htmlFor="email">Email</label></h6>
-						<input id="email" className="applicant-input" 
+						<textarea
+							id="email"
+							className="applicant-input" 
 							onChange={handleFormChange}
+							rows={1}
 						/>
 					</div>
 				</div>
-				<div className="applicant-attributes">
+				<div className="applicant-options">
 				<h5>Select your skills and values</h5>
-						<h5>Skills</h5>
-					<div className="attribute-box">
-						{makeTags(tags.skills)}
+						<h6>Skills</h6>
+					<div className="option-box">
+						{makeTags(tags.skills, 'skills')}
 					</div>
-						<h5>Values</h5>
-					<div className="attribute-box">
-						{makeTags(tags.values)}
+						<h6>Values</h6>
+					<div className="option-box">
+						{makeTags(tags.values, 'values')}
 					</div>
 				</div>
 			</div>
-				<button className="cta-button">create your profile</button>
+			<button
+				className="cta-button"
+				style={{ margin: '1em auto' }}
+				onClick={createNewApplicant}
+			>
+				create your profile
+			</button>
+			{successfulPost &&
+				<Redirect
```

Comment:
A great start! We may want to come back and review this; this way requires an additional fetch on the next page which we can prevent by passing props from this component. I think this is great for starting out though and keeping our workflow tight.

Reply:
Agreed. We should be passing the data returned from the successful POST into Applicant and using that to display. For readability, I'd prefer to have successfulPost be a boolean and applicant or newApplicant or something that holds the response.